app/views/layouts/_attachments.html.haml

```diff
@@ -0,0 +1,2 @@
+- record.send(attachment_name).each do |attachment|
```

Comment:
I assumed we would do a helper for this as well instead of a layout

Reply:
done